backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/webservice/schema/types/AcceptingStore.kt

```diff
@@ -1,25 +1,27 @@
 package app.ehrenamtskarte.backend.stores.webservice.schema.types
 
-import app.ehrenamtskarte.backend.stores.webservice.dataloader.CATEGORY_LOADER_NAME
-import app.ehrenamtskarte.backend.stores.webservice.dataloader.CONTACT_LOADER_NAME
-import app.ehrenamtskarte.backend.stores.webservice.dataloader.PHYSICAL_STORE_BY_STORE_ID_LOADER_NAME
+import app.ehrenamtskarte.backend.common.webservice.fromEnvironment
+import app.ehrenamtskarte.backend.stores.webservice.dataloader.categoryLoader
+import app.ehrenamtskarte.backend.stores.webservice.dataloader.contactLoader
+import app.ehrenamtskarte.backend.stores.webservice.dataloader.physicalStoreByStoreIdLoader
 import graphql.schema.DataFetchingEnvironment
 import java.util.concurrent.CompletableFuture
 
+@Suppress("unused")
 data class AcceptingStore(
     val id: Int,
     val name: String?,
     val description: String?,
     val contactId: Int,
-    val categoryId: Int
+    val categoryId: Int,
 ) {
 
-    fun contact(dataFetchingEnvironment: DataFetchingEnvironment): CompletableFuture<Contact> =
-        dataFetchingEnvironment.getDataLoader<Int, Contact>(CONTACT_LOADER_NAME).load(contactId)
+    fun contact(environment: DataFetchingEnvironment): CompletableFuture<Contact> =
+        contactLoader.fromEnvironment(environment).load(contactId).thenApply { it!! }
 
-    fun category(dataFetchingEnvironment: DataFetchingEnvironment): CompletableFuture<Category> =
-        dataFetchingEnvironment.getDataLoader<Int, Category>(CATEGORY_LOADER_NAME).load(categoryId)
+    fun category(environment: DataFetchingEnvironment): CompletableFuture<Category> =
+        categoryLoader.fromEnvironment(environment).load(categoryId).thenApply { it!! }
 
-    fun physicalStore(dataFetchingEnvironment: DataFetchingEnvironment): CompletableFuture<PhysicalStore?> =
-        dataFetchingEnvironment.getDataLoader<Int, PhysicalStore?>(PHYSICAL_STORE_BY_STORE_ID_LOADER_NAME).load(id)
+    fun physicalStore(environment: DataFetchingEnvironment): CompletableFuture<PhysicalStore?> =
+        physicalStoreByStoreIdLoader.fromEnvironment(environment).load(id)
```

Comment:
It seems you forgot sth. here.
```suggestion
        physicalStoreByStoreIdLoader.fromEnvironment(environment).load(id).thenApply { it!! }
```

Reply:
Actually not: We designed our API to support AcceptingStores that do not have a PhyiscalStore, (so this is a 0..1 connection). Hence, the `physicalStore` function here should return a nullable value.